crates/matrix-sdk-base/src/store/memory_store.rs

```diff
@@ -290,10 +290,8 @@ impl StateStore for MemoryStore {
 
         if let Some(s) = &changes.sync_token {
             *self.sync_token.write().unwrap() = Some(s.to_owned());
-            trace!("assigned sync token");
```

Comment:
I wouldn't mind having those left in, after all they are trace logs. Feel free to disagree if you find them spammy.

Reply:
I don't mind really. They aren't sufficiently spammy that it'd be a problem to keep them in.

Reply:
Added them back.